src/test/resources/feature/checkout2.feature

```diff
@@ -0,0 +1,69 @@
+Feature: complete order
+
+  Background: User is logged in
+
+  Background: User has items in the cart
```

Comment:
Please use single Background for both steps you need there:
Background:
  When User is logged in
  And User has items in the cart

Reply:
Done